src/translations/en/about.mdx

```diff
@@ -1,4 +1,4 @@
-This means its source code can be freely reused and built upon, as long as improvements are made available to the community under the same conditions. This also means that decisions on the evolution of the product will be taken in an increasingly democratic manner as the community that builds it will grow.
+This means its source code can be freely reused and built upon, as long as improvements are made available to the community under the same conditions. This also means that decisions on the evolution of the product will be taken in an increasingly democratic manner as the community that builds it will grow. Its [budget](/budget) is public and transparent.
```

Comment:
FYI This link will have the effect to reload the page.

If you want it not to, you should use a `Link` component

Reply:
Thanks for the clarification. Fixed with https://github.com/ambanum/OpenTermsArchive.org/pull/154/commits/c301b16dcc1a37a6d39f7282bcb39e146172b580